cove_iati/lib/process_codelists.py

```diff
@@ -102,7 +122,7 @@ def is_int(value):
 
 
 def invalid_codelist_values(codelist_values, filename, source_map=None):
-    path_to_codelist = create_path_to_codelists(codelist_values)
+    #path_to_codelist = create_path_to_codelists(codelist_values)
```

Comment:
Is this meant to be removed totally, or should it be commented back in?

I'm not sure we have a normal practice of leaving commented code.

Reply:
yes that should be removed

Reply:
It has now